src/command.rs

```diff
@@ -49,6 +49,131 @@ macro_rules! delegate {
     }};
 }
 
+/// If a command is `OverSsh` then it can be executed over an SSH session.
+/// Primarily a way to allow `std::process::Command` to be turned directly into an `openssh::Command`.
+pub trait OverSsh {
+    /// Given a session, return a command that can be executed over that session.
+    /// 
+    /// **Note**: The command to be executed on the remote machine does not include 
+    /// any environment variables or the current directory. They must be
+    /// set explicitly, if desired.
+    fn over_session<'session>(&self, session: &'session Session) -> crate::Command<'session>;
+}
+
+impl OverSsh for std::process::Command {
+    /// Given a session, convert a `std::process::Command` into an `openssh::Command` 
+    /// that can be executed over that session.
+    /// **Note**: The command to be executed on the remote machine does not include 
+    /// any environment variables or the current directory. They must be
+    /// set explicitly, if desired.
+    /// ```no_run
+    /// # #[tokio::main(flavor = "current_thread")]
+    /// async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    ///     use std::process::Command;
+    ///     use openssh::{Session, KnownHosts, OverSsh};
+
```

Comment:
```suggestion
    ///
```

Having a blank line like this would not show up in the example on docs.rs

It has to be prefixed with `///`.